package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "workflow-status-slack-notification",
+  "version": "0.0.0-development",
+  "description": "Github Action workflow failure/success notification.",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">=14"
```

I suggest we just start with Node 16 since we want all our apps moved to that now anyway.

```suggestion
    "node": ">=16"
```